nvflare/fuel/utils/class_utils.py

```diff
@@ -108,18 +98,12 @@ def create_classes_table(self):
                                         not name.startswith("_")
                                         and inspect.isclass(obj)
                                         and obj.__module__ == module_name
+                                        and issubclass(obj, FLComponent)
```

Comment:
Should we do this check for FLComponent? Any Python object could be added through "name". Maybe we should ...